src/PAO_grid_transform_module.f90

```diff
@@ -54,10 +54,10 @@ module PAO_grid_transform_module
 
 contains
 
-!!****f* PAO_grid_transform_module/single_PAO_to_grid *
+!!****f* PAO_grid_transform_module/single_PAO_to_any *
```

Comment:
The name here isn't really quite right, but I can't think of an alternative!  The previous names (`single_PAO_to_grid` and `single_PAO_to_grad`) but also misleading (the second should have been `single_PAO_grad_to_grid`).

I wonder if `single_PAO_to_grid` would be better though it might confusing historically

Reply:
I don't understand the algorithm well enough to give the subroutine a descriptive name, so I'm really waiting for your input here :sweat_smile:  If you think `single_PAO_to_grid` is good, let me know.

Reply:
Possibly `single_PAO_or_grad_PAO_to_grid` which is at least descriptive! I don't think that we need the `single` part, so maybe `PAO_or_grad_PAO_to_grid`.  @tsuyoshi38 and @ayakon do you have any thoughts?

Reply:
`PAO_or_grad_PAO_to_grid` sounds reasonable to me.  (I would like to remove `_` between grad and PAO, like `PAO_or_gradPAO_to_grid`

Reply:
Technically this should be "any function that satisfies the `evaluate` interface to grid" but I suppose we can name it after the current implementations. Would `transform_to_grid` be too generic?

Reply:
I think that would risk confusion as this routine is specifically for PAOs and their derivatives (there are other functions transformed to the grid).  I've taken what @tsuyoshi38 and I agreed (`PAO_or_gradPAO_to_grid`) and pushed it.